src/main/resources/fabric.mod.json

```diff
@@ -5,7 +5,8 @@
   "name": "Disable Portals",
   "description": "Adds a simple command for disabling nether and end portals separately.",
   "authors": [
-    "SpaceClouds42"
+    "SpaceClouds42",
+    "PHELMS"
```

Comment:
You're not a primary author of this mod, you should go into the `contributors` array